### DISCUSSION TITLE
Provide workflow file name (or id) in environment variables

### DISCUSSION THREAD

**Discussion by billyvg** [community/community#10052] [Actions]
Currently we can get the name of the workflow using `GITHUB_WORKFLOW`, but there is no env var for the workflow file name. This is important in the case where we want to use the API call to [list workflow runs](https://docs.github.com/en/rest/reference/actions#list-workflow-runs). The `workflow_id` param accepts either a workflow id or a workflow file name, neither of which are provided in the context of a workflow run.

**Comment by ethomson** [comment]
Hmm, thanks for the feedback.  This seems like an oversight that we can remedy.

**Reply by tfonfara** [reply]
Is there any news on that feature? See also https://github.com/actions/runner/issues/853. Implementation of this would be highly appreciated 😊

**Reply by harrietrs** [reply]
^ seconded!!

**Reply by shaypi** [reply]
++

**Comment by robsonpeixoto** [comment]
And a `GITHUB_WORKFLOW_FILE` or `GITHUB_WORKFLOW_PATH` will be useful to configure the `paths` when setting the triggers:

```
on:
  pull_request:
    paths:
      - "**.my-extension"
      - $GITHUB_WORKFLOW_FILE
```
